File: ios/Keyboard/KeyboardModule.m
//
//  KeyboardModule.m
//  Keyboard
//
//  Created by Nomads on 2021-10-22.
//

#import <Foundation/Foundation.h>
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE (KeyboardModule, NSObject)

RCT_EXTERN_METHOD(insertSticker
                  : (NSString *)stickerUrl resolver
                  : (RCTPromiseResolveBlock)resolve rejector
                  : (RCTPromiseRejectBlock)reject)

@end

File: ios/Keyboard/KeyboardModule.swift
//
//  KeyboardModule.swift
//  Keyboard
//
//  Created by Nomads on 2021-10-22.
//

import Foundation
import Keyboard


@objc(KeyboardModule)
class KeyboardModule: NSObject {
  let viewController: KeyboardViewController

  static func moduleName() -> String! {
    "RCTKeyboard"
  }

  static func requiresMainQueueSetup() -> Bool {
    false
  }
  
  init(viewController: KeyboardViewController) {
    self.viewController = viewController
  }

  /**
   Insert a sticker into the active conversation.
   */
 

      // After inserting a sticker, request that the extension collapse back to

}

